tests/acceptance/features/apiCollaboration/checkFileInfo.feature

```diff
@@ -588,3 +588,14 @@ Feature: check file info with different wopi apps
       | view  | true          | false          | false           |
       | read  | false         | false          | false           |
       | write | false         | true           | true            |
+
+
+  Scenario Outline: try to get file info using invalid file id with office suites
+    Given user "Alice" has uploaded file with content "hello world" to "/textfile0.txt"
+    When user "Alice" tries to check the information of file "textfile0.txt" of space "Personal" using office "<office-suites>" with invalid file-id
+    Then the HTTP status code should be "401"
```

Comment:
IMO, checking the information with invalid id means the resource should not be found with that id (meaning status code `404` or `400`)? Or should it be unauthorized; as indicated by the status code `401` above?

Reply:
request uses access tokens of valid files so maybe source code validate access token and throws 401 
Source of HTTP status code: https://github.com/owncloud/ocis/pull/9800/files#diff-b8e0ed53726850dccb8028a70292d46a01bc86b476f289ed3d6a9758d9704179R112 .

Reply:
upto my understanding of source code
they match the file-id inside the access token with the file-id provided in the URL. The source code doesn't check whether the file-id is valid or not. Because file-id validation is done while getting the access token. it just validates file-id encoded in the access token is same as the file-id in URL or not. Seems logical to me.

Reply:
let's keep it as it is. since we are doing request to wopi server not the ocis server.